src/Symfony/src/OtelSdkBundle/DependencyInjection/ConfigMappings.php

```diff
@@ -26,4 +26,8 @@ interface ConfigMappings
         Configuration::OTLP_GRPC_EXPORTER => SpanExporters::OTLP_GRPC,
         Configuration::ZIPKIN_TO_NEWRELIC_EXPORTER => SpanExporters::ZIPKIN_TO_NEWRELIC,
```

Comment:
anything related to NewRelic can be removed - we dropped their custom exporters as they now fully support otlp